R/annotation.R

```diff
@@ -31,6 +31,7 @@ annotate_ssm_blacklist = function(mutations_df,
                                   blacklist_file_template,
                                   drop_threshold = 4,
                                   return_blacklist = FALSE,
+                                  use_curated_blacklist = FALSE,
```

Comment:
Currently there's an issue with the logic here because I think the curated blacklist will be applied after the regular blacklist and the contents of `dropped` will be clobbered. It would probably be better if you set this up such that using the curated blacklist caused the earlier code involving the other blacklists to be skipped. e.g.

```
#run any code that is needed in all cases, e.g. setting projection from genome_build. 
if( ! use_curated_blacklist{
 do all the regular blacklist stuff and create/populate the two data frames to return them later
}else{
 do your new blacklist stuff and create/populate the the two data frames to return them outside this block
}
#code that runs in all cases:
if(invert){
    return(dropped)
  }
  return(mutations_df)
```